notebooks/data_processing.ipynb

```diff
@@ -0,0 +1,795 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    met_data_path = '/home/reddyanugu/aqmsp/projects/better_interpol/notebooks/meteorological_data.nc'
</pre><p>This file shouldn't have had locations of sensors present within it. Perhaps it was prepared somewhere else first?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/patel-zeel/kdd24/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>